_articles/account/biometrics.md

```diff
@@ -110,7 +110,7 @@ Unlock with Biometrics is supported for Extensions on **Chromium-based** browser
 
 To enable Unlock with Biometrics for your Browser Extension:
 
-{% callout success %}Biometrics must be enabled in your Desktop App before proceeding. Additionally, **if you're using Safari**, you can skip straight to **Step 4**.{% endcallout %}
+{% callout success %}Biometrics (Windows Hello or Touch Id) must be enabled in your Desktop App before proceeding. If you don't see the Windows Hello option in your Desktop application, please see the troubleshooting tips in the Desktop article. Additionally, **if you're using Safari**, you can skip straight to **Step 4**.{% endcallout %}
```

Comment:
@tgreer-bw IMO we can directly reference the tip rather than referring them elsewhere if you want, something like `If you don't see the Windows Hello option in your Desktop app, you may need to [install the Microsoft Visual C++ Redistributable](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0){:target="\_blank}.`